### DISCUSSION TITLE
WIki/documentation?

### DISCUSSION THREAD

**Discussion by squelch41** [frntc/RAD#28] [Q&A] [answered]
Hi,
Is there a wiki or documentation about the menu anywhere?
Struggling to understand how the REU imaging works - eg when use GEOS, can save an image but when try and load it, just goes to basic screen and cant run the image again.
Appears in HEX editor that GEOS is in the image.

**Comment by frntc** [comment]
You mean when you re-mount the saved REU-image and exit the menu you end up in Basic? That would be correct, as you'd have to boot GEOS again. Did I get you correctly?

**Reply by squelch41** [reply]
Yes, I think I misunderstood how the imaging works. I thought you could re-start the application eg goes with a SYS([memory address] command but I've obviously got the wrong end of the stick :)

**Reply by frntc** [reply, marked as answer]
that would be convenient, but unfortunately that's not possible (a REU does not bank-in memory anywhere, e.g. to jump to it using a SYS command)

**Comment by squelch41** [comment]
:)

Thanks for creating this project btw - it's awesome
